Interface/AddOns/nMainbar/modules/short/bags.lua

```diff
@@ -0,0 +1,82 @@
+
+local _, nMainbar = ...
+local cfg = nMainbar.Config
+
+if (not cfg.MainMenuBar.shortBar) then
+    return
+end
+
+--Binding Info
+BINDING_HEADER_NBAGS = "nBags"
+BINDING_NAME_NBAGS_TOGGLE = "Toggle Bags"
+
+-- Saved Variable Setup
+if shown == nil then
+    shown = false
+end
+
+-- Clear frame position.
+MainMenuBarBackpackButton:ClearAllPoints()
+CharacterBag0Slot:ClearAllPoints()
+CharacterBag1Slot:ClearAllPoints()
+CharacterBag2Slot:ClearAllPoints()
+CharacterBag3Slot:ClearAllPoints()
+
+-- Set new frame position.
+MainMenuBarBackpackButton:SetPoint('BOTTOMRIGHT', WorldFrame, 'BOTTOMRIGHT',0, 0)
+CharacterBag0Slot:SetPoint('RIGHT', MainMenuBarBackpackButton, 'LEFT',0, 0)
+CharacterBag1Slot:SetPoint('RIGHT', CharacterBag0Slot, 'LEFT',0, 0)
+CharacterBag2Slot:SetPoint('RIGHT', CharacterBag1Slot, 'LEFT',0, 0)
+CharacterBag3Slot:SetPoint('RIGHT', CharacterBag2Slot, 'LEFT',0, 0)
+
+-- Set !BeautyBorder Settings
+if IsAddOnLoaded('!Beautycase') then
+    MainMenuBarBackpackButton:CreateBeautyBorder(8)
+    CharacterBag0Slot:CreateBeautyBorder(8)
+    CharacterBag1Slot:CreateBeautyBorder(8)
+    CharacterBag2Slot:CreateBeautyBorder(8)
+    CharacterBag3Slot:CreateBeautyBorder(8)
+end
```

Comment:
I'm thinking this isn't needed anymore? All buttons should more or less be automatically skinned since we override the texture in `Interface/Buttons`.